<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(204, 168, 233);
color: rgb(245, 235, 235);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(9, 9, 9);</string>
    </property>
    <property name="text">
     <string>Device Tracking System</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>391</width>
      <height>242</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <property name="verticalSpacing">
      <number>40</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="0" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>50</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(9, 9, 9);</string>
         </property>
         <property name="text">
          <string>Image :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="select_img_button">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 190, 240);
color: rgb(9,9,9);
padding : 15px 40px;
border-radius:8px;
border: 1px solid black

</string>
         </property>
         <property name="text">
          <string>Select Image</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>50</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 190, 240);
color: rgb(9,9,9);
padding : 15px 20px;
border-radius:8px;
border: 1px solid black;

</string>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generate_button">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 190, 240);
color: rgb(9,9,9);
padding : 15px 20px;
border-radius:8px;
border: 1px solid black

</string>
         </property>
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>407</width>
      <height>464</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_5">
     <property name="verticalSpacing">
      <number>40</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="0" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>40</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(9, 9, 9);</string>
         </property>
         <property name="text">
          <string>Device Name :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="device_name_combobox">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    background-color: rgb(195, 190, 240);
    color: rgb(9, 9, 9);
    padding: 12px 60px; /* Adjust padding as needed */
    border-radius: 8px;
    border: 1px solid black;
}

QComboBox::item {
    text-align: left; /* Align text to the left */
    padding-top: 10px; /* Adjust the padding to center text vertically */
    padding-bottom: 10px; /* Adjust the padding to center text vertically */
}
</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(9, 9, 9);</string>
         </property>
         <property name="text">
          <string>Serial No. :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="Serial_No_combobox">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    background-color: rgb(195, 190, 240);
    color: rgb(9, 9, 9);
    padding: 12px 60px; /* Adjust padding as needed */
    border-radius: 8px;
    border: 1px solid black;
}

QComboBox::item {
    text-align: left; /* Align text to the left */
    padding-top: 10px; /* Adjust the padding to center text vertically */
    padding-bottom: 10px; /* Adjust the padding to center text vertically */
}
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QFormLayout" name="formLayout_2">
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>40</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item row="0" column="1">
        <widget class="QComboBox" name="assignedto_combobox">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    background-color: rgb(195, 190, 240);
    color: rgb(9, 9, 9);
    padding: 12px 60px; /* Adjust padding as needed */
    border-radius: 8px;
    border: 1px solid black;
}

QComboBox::item {
    text-align: left; /* Align text to the left */
    padding-top: 10px; /* Adjust the padding to center text vertically */
    padding-bottom: 10px; /* Adjust the padding to center text vertically */
}
</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(9, 9, 9);</string>
         </property>
         <property name="text">
          <string>Location :       </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="location_combobox">
         <property name="styleSheet">
          <string notr="true">QComboBox {
    background-color: rgb(195, 190, 240);
    color: rgb(9, 9, 9);
    padding: 12px 60px; /* Adjust padding as needed */
    border-radius: 8px;
    border: 1px solid black;
}

QComboBox::item {
    text-align: left; /* Align text to the left */
    padding-top: 10px; /* Adjust the padding to center text vertically */
    padding-bottom: 10px; /* Adjust the padding to center text vertically */
}
</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(9, 9, 9);</string>
         </property>
         <property name="text">
          <string>Assign to :        </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="update_user_button">
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(195, 190, 240);
color: rgb(9,9,9);
padding : 15px 20px;
border-radius:8px;
border: 1px solid black;


</string>
         </property>
         <property name="text">
          <string>Update Details</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="photolabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align:centre;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
